pkg/apply/utils.go

```diff
@@ -109,3 +109,24 @@ func getHostArch(sshClient ssh.Interface) func(string) v2.Arch {
 func GetHostArch(sshClient ssh.Interface, ip string) string {
 	return string(getHostArch(sshClient)(ip))
 }
+
+func GetImagesDiff(current, desired []string) []string {
```

Comment:
IMO, we should keep the order of images, sometimes `image 1` might depends on `image 2`

Reply:
> IMO, we should keep the order of images, sometimes `image 1` might depends on `image 2`

Yes, it's better to keep the order while diff.

Reply:
do I need to consider duplicate images in the Clusterfile?

Reply:
> IMO, we should keep the order of images, sometimes `image 1` might depends on `image 2`

done